UpdatedTicketHubDDL.sql

```diff
@@ -0,0 +1,173 @@
+CREATE TABLE User (
+    UserID INT PRIMARY KEY AUTO_INCREMENT,
+    Username VARCHAR(50) UNIQUE,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Birthdate DATE,
+    Password VARCHAR(100),
+    Email VARCHAR(100) UNIQUE
+);
```

Comment:
Missing type of user (E.g. buyer/seller), missing middle name field. Also all name values should be 16 characters max -  same with username.

Reply:
- I think FirstName, MiddleName, LastName should be a composite key of the UserName for query simplicity. 
- As Tatum mentioned, password should have proper length constraint.
- I don't think we are supposed to have 'UserId' in this table